decorator.rb

```diff
@@ -0,0 +1,12 @@
+require_relative 'nameable'
+
```

Comment:
- [ ] Please add the correct association between the nameable class and the decorator class using the `attr_accessor` 🙏

Reply:
> * [ ]  Please add the correct association between the nameable class and the decorator class using the `attr_accessor` 🙏

@MahmoudBakr23  I am not understood, could you brief it?

Reply:
> * [ ]  Please add the correct association between the nameable class and the decorator class using the `attr_accessor` 🙏

I am working on decorator class, and I tried to put based on the solution.

Reply:
@MahmoudBakr23 fixed.